webapp/channels/src/components/admin_console/push_settings.jsx

```diff
@@ -53,6 +56,21 @@ export default class PushSettings extends AdminSettings {
             }
         }
 
+        if (id === 'pushNotificationServerLocation') {
+            switch (value) {
+            case PUSH_NOTIFICATIONS_LOCATION_US:
```

Comment:
nit: even when JS is optimized for switch case statements, for such simple case conditions like this one that doesn't even use a default, it is IMO for simplicity and readability to create a dictionary with the values and then assign the state directly:

```
const pushDic = {
    [PUSH_NOTIFICATIONS_LOCATION_US]: Constants.MHPNS_US,
    [PUSH_NOTIFICATIONS_LOCATION_DE]: Constants. MHPNS_DE,
 };
```
  ...
  
```
this.setState({
   pushNotificationServer: pushDic[value],
});
```

Reply:
i agree on @pvev but above switch state reads a lot easier to me

Reply:
@M-ZubairAhmed To which switch statement are you referring? The one newly added by me? Or the `if` `else` chain in https://github.com/mattermost/mattermost/blob/1de37de08093b4432f6b2f5de7ea427f0f48d323/webapp/channels/src/components/admin_console/push_settings.jsx#L45-L59?

Reply:
@M-ZubairAhmed Could you please help me clarify your comment?

Reply:
Sorry missed this comment, i meant to say the switch statement which you added reads more clearly to be.